.github/workflows/label-pull-requests.yml

```diff
@@ -0,0 +1,26 @@
+name: Pull request labeler
+
+# FIXME: Consider adding the `issue_comment` event to change labels based on generic, non-review pull request comments.
+#        Consider the trade off of how spammy it can be (one CI run per comment) and how useful it would be to have.
+#        Consider alternatives to `issue_comment`.
+on:
+  pull_request_target:
+    types: [opened, reopened, converted_to_draft, ready_for_review, synchronize, edited, review_requested, closed]
+  pull_request_review:
+    types: [submitted, edited, dismissed]
+
+jobs:
+  label_pull_request:
+    runs-on: ubuntu-22.04
+    if: always() && github.repository == 'SerenityOS/serenity'
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Label pull request
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.BUGGIEBOT_TOKEN }}
+          script: |
+            const script = require('./Meta/label-pull-requests.js')
```

Comment:
I think this script file is missing.

Reply:
Yes, Git is having a real good time today.